sci-geosciences/gpxlab/gpxlab-0.5.0.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="Program to show and manipulate GPS tracks"
+DESCRIPTION="GPXLab is an application to display and manage GPS tracks \
+previously recorded with a GPS tracker."
+HOMEPAGE="https://bourgeoislab.wordpress.com/gpxlab/"
+COPYRIGHT="2018 Frédéric Bourgeois"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/BourgeoisLab/GPXLab/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="430672eaf0c76e1f7f43cf2c117e63d818ac7e444acceefe6a0ef5a2f7b3d7f4"
+SOURCE_DIR="GPXLab-$portVersion"
+PATCHES="gpxlab-$portVersion.patchset"
+ADDITIONAL_FILES="gpxlab.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gpxlab$secondaryArchSuffix = $portVersion
+	app:GPXLab = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:lrelease$secondaryArchSuffix
+	cmd:make
+	cmd:qmake$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	lrelease GPXLab/GPXLab.pro
+	qmake GPXLab.pro
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/GPXLab/translations
+	cp bin/GPXLab $appsDir/GPXLab/
+	cp -r GPXLab/locale/*.qm $appsDir/GPXLab/translations
```

Comment:
I think it would be nice to:
* install the app as `$appsDir/GPXLab` instead of `$appsDir/GPXLab/GPXLab`
* install the translations in `$appsDir/gpxlab/translations/` instead of `$appsDir/GPXLab/translations/`

But I guess this require to modify some parameters in `GPXLab.pro`.

I'm bringing this subject because having the GPXLab runtime in `$appsDir` would make it easier to launch the app from the command line, as `/boot/system/apps` is in the default `$PATH`, while `/boot/system/apps/GPXLab` is not.

But I know most contributors are ok with your layout, and some even prefer it, so I'm ok with it too.
That said, if you would like to investigate and use the layout I'm suggesting then I think it would also be fine for everyone :)

Reply:
Yep. (The following links are all in 2 of our wiki pages:)
* [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)
* [`COPYRIGHT`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#copyright)
* [`HOMEPAGE`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#homepage)
* [HaikuPorter-Guidelines#archive-name](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name)
* [Using the commandBinDir User Variable](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-commandbindir-user-variable)

Reply:
Oops, I just realized you were asking for the filesystem layout, not the the recommended layout for a recipe.
For the filesystem hierarchy I don't know if we have any doc other than the specs of the **_hpkg_** package format.